src/PrestaShopBundle/Resources/views/Admin/Module/_partials/_modules_modal_read_more.html.twig

```diff
@@ -1,15 +1,15 @@
-<div id="module-modal-read-more-{{module.name}}{{ additionalModalSuffix|default('') }}" class="modal modal-vcenter fade" role="dialog">
+<div id="module-modal-read-more-{{module.attributes.get('name')}}{{ additionalModalSuffix|default('') }}" class="modal modal-vcenter fade" role="dialog">
     <div class="modal-dialog module-modal-dialog">
```

Comment:
How about a `ModulePresenter` able to return exploitable representation in front ?